sections/components.xml

```diff
@@ -1626,58 +1657,80 @@ class Dispatcher extends \Joomla\CMS\Dispatcher\ComponentDispatcher
       of the component using a new XML element under the
       <code>&lt;extension&gt;</code> root element:</para>
 
-      <programlisting language="xml">&lt;namespace path="src"&gt;My\Namespace\Prefix&lt;/namespace&gt;</programlisting>
+      <programlisting language="xml">&lt;namespace path="src"&gt;MyComponentNamespace&lt;/namespace&gt;</programlisting>
 
       <para>The <code>path</code> attribute tells Joomla which subdirectory of
       your extension holds the PSR-4 of your extension's PHP files. It is best
       practice to name it <filename>src</filename> but <emphasis>you don't
       have to</emphasis>. I always assume you are using
       <filename>src</filename>.</para>
 
-      <para>The text inside the XML element, <code>My\Namespace\Prefix</code>
+      <para>The text inside the XML element, <code>MyComponentNamespace</code>
       in our example, is the namespace prefix you will be using. Each
-      component can register its own namespace prefix without any
-      restrictions. It is <emphasis>recommended</emphasis> to use the
-      convention <code>\Company\Something</code> where <code>Company</code> is
-      your company name (or a shortened version thereof) and
-      <code>Something</code> corresponds to the <code>com_something</code>
-      extension name of your component. For example, the component
-      <code>com_example</code> by Acme, Inc could have a namespace prefix of
-      <code>\Acme\Example</code> set up in its XML manifest.</para>
+      component has its own namespace prefix, which consists of 3
+      parts:</para>
+
+      <programlisting>\&lt;vendor part&gt;\Component\&lt;component name&gt;</programlisting>
+
+      <itemizedlist>
+        <listitem>
+          <para><code>\&lt;vendor part&gt;</code>: a valid PHP namespace,
+          unique to you, your company or your project. It should not start
+          with <code>\Joomla</code>, because component namespaces starting
+          withIn <code>\Joomla</code> are reserved for core components. And it
+          should not contain <code>\Component</code>, as that would interfere
```

Comment:
Confusing syntax. Best clarify as “Component should only appear as the last part of the namespace prefix, e.g. Foo\Bar\Component but NOT Foo\Component\Bar\Component. The word Component has a special meaning in Joomla and should best be avoided in the customisable parts of your namespace prefix”. Giving a good example like you do later one is great!

Reply:
No: the component name must be namespace part immediately after the FIRST occurrence of "\Component". 

So:
* NOT: “Component should only appear as the last part of the namespace prefix, e.g. Foo\Bar\Component"
* but "Foo\Component\Bar\Component", although not advisable, would not be a broblem.

I'll try to make it as clear as possible.